refactor!: make modules the main focus, move files into module folders

xynydev opened the pull request:
- Less clutter overall
- A lot of the bling was unmaintained and underutilized
- This part of the transition being discussed here: https://github.com/ublue-os/main/issues/476

This is a breaking change that removes features, features that were unmaintained and undocumented (or badly documented).

Commit:
chore: remove remaining bling justfile

Commit:
chore: remove static font files (use fonts module instead)

Commit:
chore: remove ublue nix installers in path
superseded by just setup-nix

Commit:
refactor: move default-flatpaks files to module directory

Commit:
refactor: move bling files to bling module

Commit:
chore: remove deprecated/unfunctional bling

Commit:
docs: update bling module readme

Commit:
chore: remove nix-app-icons.sh
i couldn't find anything that uses this
and i'm not sure it's even needed anymore
and it's totally unmaintained

Commit:
chore: remove stray bits from containerfile

Commit:
chore: remove fetch.sh
- i just noticed this is where the container tools for the container tools bling were coming from, but i already removed that...
- ig this wasn't the only place to get container tools
- the devpod bin wasn't used anywhere either and was locked to a specific version (that no-one maintained to update)

Commit:
chore: remove devpod module
- tries to install nonexistant rpm

Commit:
docs: update README to reflect current state of repo
- also, there should only be one h1 per page

Commit:
feat: include files in yafti module

Commit:
fix: rpm builds failing?

Commit:
fix: rpm build license error pt2?

Commit:
fix: rpm build licence error pt3?

Commit:
fix: rpm build license error pt4?

Commit:
chore: disable the wallpaper builds for now
this is a nightmare to poke around in blindly, lets see where #104 goes

fiftydinar commented:
Looks very good.

Maybe we can remove ublue-os backgrounds rpm related stuff here too?

xynydev commented:
> Maybe we can remove ublue-os backgrounds rpm related stuff here too?

I was thinking that'd be a later PR but I can do it now too, sure.

fiftydinar commented:
https://github.com/ublue-os/bling/blob/refactor/modules/bling/bling.sh#L12

This one needs to be: 
`export BLING_DIRECTORY="/tmp/modules/bling"`

Commit:
chore: remove ublue-os-wallpapers

Commit:
chore: remove build steps in containerfile
- also removes ublue-update rpm, this isn't used in the installer, the copr repo is

xynydev commented:
> This one needs to be: `export BLING_DIRECTORY="/tmp/modules/bling"`

Good catch! This doesn't need to be at all actually, the `$MODULE_DIRECTORY`var (supplied by build.sh, but also manually set just in case) is used instead.

Commit:
chore: remove BLING_DIRECTORY envvar
superseded by use of MODULE_DIRECTORY

fiftydinar commented:
I would approve
